### DISCUSSION TITLE
Charts size customization

### DISCUSSION THREAD

**Discussion by ced75** [evidence-dev/evidence#2715] [Ideas]
### Description

I haven't found if it is possible to customize chart size.
My purpose is to increase a Line Chart height.
I have tried with ChartAreaHeight property, but it doesn't seem to have any effect.

I don't know if it is already possible...

**Comment by ebardelli** [comment]
Most charts have the `chartAreaHeight` property to make it longer.

**Reply by ced75** [reply]
Thanks for your reply.
As I said, I have found this property, but increasing its value seems to have no effect.

**Reply by ebardelli** [reply]
Which chart are you working with? I think it's not implemented for some.

**Reply by ced75** [reply]
A LineChart with multiple lines.
I don't know where was the mistake, but a new test seems to work fine now...
